packages/yoco/svg/catalog.svg

```diff
@@ -0,0 +1,24 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<!-- <rect x="7" y="6" width="16" height="16" fill="white" />
+	<rect x="7" y="25" width="16" height="16" fill="white" /> -->
+
+	<mask id="path-1" fill="white" maskUnits="userSpaceOnUse" maskContentUnits="userSpaceOnUse">
+		<rect x="7" y="6" width="16" height="16" rx="2" />
+	</mask>
+	<mask id="path-2" fill="white" maskUnits="userSpaceOnUse" maskContentUnits="userSpaceOnUse">
+		<rect x="7" y="25" width="16" height="16" rx="2" />
+	</mask>
+	<g stroke="black" stroke-width="8">
+		<rect x="7" y="6" width="16" height="16" rx="2" mask="url(#path-1)" />
+		<rect x="7" y="25" width="16" height="16" rx="2" mask="url(#path-2)" />
+	</g>
+
+	<g stroke="black" stroke-linecap="round">
+		<line x1="27" y1="8" x2="39" y2="8" stroke-width="4" />
+		<line x1="27" y1="27" x2="39" y2="27" stroke-width="4" />
+		<line x1="27" y1="14" x2="39" y2="14" stroke-width="4" />
+		<line x1="27" y1="33" x2="39" y2="33" stroke-width="4" />
+		<line x1="27" y1="20" x2="33" y2="20" stroke-width="4" />
+		<line x1="27" y1="39" x2="33" y2="39" stroke-width="4" />
+	</g>
+</svg>
```

Comment:
Your editorconfig seems to be malfunctioning. There’s no final newline here which is invalid on UNIX-based systems